vm-builder/main.go

```diff
@@ -348,6 +369,8 @@ func main() {
 		{"vmstart", scriptVmStart},
 		{"vmshutdown", scriptVmShutdown},
 		{"inittab", scriptInitTab},
+		{"agetty-init-amd64", scriptAgettyInitAmd64},
+		{"agetty-init-arm64", scriptAgettyInitArm64},
```

Comment:
IIUC these are not used in the dockerfile, right?

Reply:
that's correct, thanks for pointing this out!